pkg/thing/interactors/auth_thing.go

```diff
@@ -11,15 +13,16 @@ func (i *ThingInteractor) Auth(authorization, id string) error {
 
 	_, err := i.thingProxy.Get(authorization, id)
 	if err != nil {
-		i.logger.Error(err)
-		err = i.clientPublisher.SendAuthStatus(id, err)
-		return err
+		sendErr := i.clientPublisher.SendAuthStatus(id, err)
+		if sendErr != nil {
+			return fmt.Errorf("error getting thing metadata: %v: %w", sendErr, err)
+		}
+		return fmt.Errorf("error getting thing metadata: %w", err)
```

Comment:
Is this change related to grouping business logic errors?

Reply:
You're right! Split it in another commit